test/Types.hs

```diff
@@ -45,6 +45,13 @@ import Language.Haskell.TH.Lib (starK)
 import Data.Kind
 #endif
 
+starT :: Kind
+#if MIN_VERSION_template_haskell(2,8,0)
+starT = StarT
+#else 
+starT = StarK
+#endif
```

Comment:
Instead of defining your own thing using CPP, just use [`starK`](https://hackage.haskell.org/package/template-haskell-2.20.0.0/docs/Language-Haskell-TH-Lib.html#v:starK) from `Language.Haskell.TH.Lib`, which is fully backwards-compatible.

Reply:
Great spot! I'll use that function instead